src/config/external-applications.md

```diff
@@ -0,0 +1,51 @@
+# External Applications
+
+## Programming Languages
+
+> The Void repositories have a number of python and lua packages. If possible,
+> try to install packages from the Void repos using xbps-install or use xbps-src
+> to package your python or lua application. Packaging your application with
+> xbps-src will keep your system clean. If you send your package upstream it
+> will allow other Void users to benifit and allow them to help you maintain
+> your package. To learn how to package for xbps-src see the
+> [manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+> and [contributing
+> instructions](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
+
+To keep your system lean, Void splits packages and their devel packages. If you
+install a program via the languages package manager (pip, gem, etc.) or compile
+via source you may need to install the programming language's devel package to
+ensure your program builds.
+
+| Language | Package Manager                | Void Package  |
+|----------|--------------------------------|---------------|
+| Python   | pip, anaconda, virtualenv, etc | python3-devel |
+| Ruby     | gem                            | ruby-devel    |
+| lua      | luarocks                       | lua-devel     |
```

Comment:
Use code blocks: `python3-devel`, `ruby-devel` and `lua-devel`. I'm not completely against making a section for Python 2 and one for Python 3, either.

Reply:
I added both your suggestions